zarr/v3/codecs/__init__.py

```diff
@@ -31,35 +30,29 @@
     from zarr.v3.codecs.crc32c_ import Crc32cCodecMetadata
 
 
+def _find_array_bytes_codec(
+    codecs: Iterable[Tuple[Codec, ArraySpec]]
+) -> Tuple[ArrayBytesCodec, ArraySpec]:
+    for codec, chunk_metadata in codecs:
+        if isinstance(codec, ArrayBytesCodec):
+            return (codec, chunk_metadata)
```

Comment:
@normanrz - can you comment on whether we have a check elsewhere to make sure we never have two `ArrayBytesCodec`s in the array metadata object?

Reply:
Yes, that is in here https://github.com/normanrz/zarr-python/blob/codecs-without-array-metadata/zarr/v3/codecs/__init__.py#L49-L51